### DISCUSSION TITLE
why is the lighthouse deck blinking orange

### DISCUSSION THREAD

**Discussion by crazy-interns** [bitcraze/discussions#1566] [Q&A]
my lighthouse deck keeps blinking orange. it can sense calibration and recieving data but nothing else. i already set the base station channel. i have 1 v2 base station

**Comment by tobbeanton** [comment]
I think that is because the deck does not have any base station geometry. Have you done that [step](https://www.bitcraze.io/documentation/tutorials/getting-started-with-lighthouse/#estimate-the-geometry) in the getting started guide?

**Reply by crazy-interns** [reply]
i did it. i dont know why geo/calib is not working. it keeps blinking

**Reply by crazy-interns** [reply]
i have 5 drones, 2 of their deck is not working, one is fully red, one is blinking orange. how do i fix it? how do i estimate geometry with only 1 lighthouse v2? i need atleast 2 for the link

**Reply by knmcguire** [reply]
Geometry estimation simple needs to be used for a single basestation: https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/lighthouse_tab/#basestation-geometry-management
